### ISSUE TITLE
Need a mechanism to cleanup session for which we never received a stop

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#5339]
**Describe the bug**
We need to have a mechanism to delete RADIUS accounting sessions for which we never received a stop. Otherwise these will rot forever in the table.

The logic in radacct was that if a session was inactive for more than the window of deletion, then it would be marked as closed and then deleted by the cleanup job. That logic should be fine for bandwidth_accounting AFAIK using the last_updated of the latest bucket to know if a session is still active.

**Comment by julsemaan** [comment]
This doesn't work 

My package version:
packetfence-9.3.9-20200409134049.134439865.0007.devel.el7.x86_64

What I did:
1. Insert row that is older than 2 days
```
insert into bandwidth_accounting VALUES(445146671383899, 1, "2020-04-01 03:05:00", "radius", 1, 1, "94:db:c9:38:85:5b", 1, "2020-04-01 03:05:00", 2);
```
2. Run `bin/pfcmd pfmon bandwidth_maintenance` (ran it a few times to be sure)
3. Row is still there
```
MariaDB [pf]> select * from bandwidth_accounting;
+-----------------+----------------------+---------------------+-------------+----------+-----------+-------------------+-----------+---------------------+-------------+
| node_id         | unique_session_id    | time_bucket         | source_type | in_bytes | out_bytes | mac               | tenant_id | last_updated        | total_bytes |
+-----------------+----------------------+---------------------+-------------+----------+-----------+-------------------+-----------+---------------------+-------------+
| 445146671383899 |                    1 | 2020-04-01 00:00:00 | radius      |        1 |         1 | 94:db:c9:38:85:5b |         1 | 2020-04-01 03:05:00 |           2 |
+-----------------+----------------------+---------------------+-------------+----------+-----------+-------------------+-----------+---------------------+-------------+
```

**Comment by julsemaan** [comment]
As @jrouzierinverse said, the frontend needs to be adjusted after https://github.com/inverse-inc/packetfence/commit/1d715153ea0c55aaf9606b359b0e7632faf03da5

**Comment by satkunas** [comment]
@julsemaan @jrouzierinverse can you describe what's needed and where?

**Comment by julsemaan** [comment]
@jrouzierinverse can probably explain his needs better than I do, just wanted to note the requirement in the issue

**Comment by satkunas** [comment]
If stops are expected within a certain time period, MariaDB events could be leveraged to automated the cleanup after a certain timeperiod has passed.

Hypothetically:
```sql
CREATE EVENT `00:00:00:00:00:00-cleanup`
    ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 1 WEEK
    DO
      DELETE FROM bandwidth_balance where mac = '00:00:00:00:00:00' and acctstoptime IS NULL;
```

**Comment by julsemaan** [comment]
From what I understood in the chat, he needs help with the frontend

**Comment by jrouzierinverse** [comment]
We could use MariaDB events but we need to test how it works in a cluster and how to manage it when a server goes down.

**Comment by jrouzierinverse** [comment]
I also figured out my mistake with the GUI side of things will be pushing in a few.

**Comment by jrouzierinverse** [comment]
This is done

**Comment by jrouzierinverse** [comment]
Fixed by b8743711940be997f6003b69d029ae47177e93e9
